libcanard/canard.c

```diff
@@ -40,8 +40,9 @@
 
 // --------------------------------------------- INTERNAL INCLUDES ----------------------------------------------
 // The internal includes are placed here after the config header is included and CANARD_ASSERT is defined.
-
-#include "_canard_cavl.h"
+#define CAVL2_T struct CanardTreeNode
+#define CAVL2_ASSERT(x) CANARD_ASSERT(x)
+#include <../lib/cavl2/cavl2.h>
```

Comment:
```suggestion
#include <cavl2.h>
```

Add `-isystem` pointing to `lib/cavl2`